src/config.rs

```diff
@@ -218,9 +205,9 @@ pub struct ScanOptions {
     pub cache_blocks: bool,
 }
 
-impl ScanOptions {
+impl Default for ScanOptions {
     /// Create a new ScanOptions with `read_level` set to [`ReadLevel::Commited`].
-    pub const fn default() -> Self {
+    fn default() -> Self {
```

Comment:
FYI: This would never be called outside of this file, instead it would fallback to macro generated implementation.